src/devices/Card/Ndef/NdefMessage.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// Create a NDEF = NFC Data Exchange Format Message class
+    /// </summary>
+    public class NdefMessage
```

Comment:
should this be internal? (I don't think you are using this anywhere in the sample)

Reply:
actually you seem to 😄 Curious - would it make sense for users to not know that they are using Ndef vs something else?

Reply:
The main notion is really Message which contains Records. So Message is needed, and yes, used in the sample